lib/cc/engine/analyzers/sexp_lines.rb

```diff
@@ -0,0 +1,32 @@
+module CC
+  module Engine
+    module Analyzers
+      class SexpLines
+        attr_reader :begin_line, :end_line
+
+        def initialize(root_sexp)
+          @root_sexp = root_sexp
+          calculate
+        end
+
+        def to_h
+          {"begin": begin_line, "end": end_line}
```

Comment:
What about leaving the hashifiying to `#format` in the other class? Feels like a "formatting" responsibility.

Then you can feel better testing the public `begin_line`/`end-line` methods that indeed need to be exposed.